go/mcap/parse.go

```diff
@@ -264,41 +264,40 @@ func ParseChunkIndex(buf []byte) (*ChunkIndex, error) {
 	}, nil
 }
 
-// ParseAttachment parses an attachment record.
-func ParseAttachment(buf []byte) (*Attachment, error) {
-	logTime, offset, err := getUint64(buf, 0)
+func parseAttachmentReader(r io.Reader) (*AttachmentReader, error) {
+	buf := make([]byte, 8)
+	crcReader := newCRCReader(r)
```

Comment:
Can we make CRC validation optional somehow?

Reply:
yeah, I can add the option to skip the computation of CRC. Actual validation of the CRC is up to the callback function the user supplies.